test/runtests.jl

```diff
@@ -25,4 +25,8 @@ if already_installed
     Conda.add("curl")
 end
 
-@test isfile(joinpath(Conda.scriptsdir("conda"), "conda" * @windows ? ".exe": ""))
\ No newline at end of file
+@test isfile(joinpath(Conda.SCRIPTDIR, "conda" * @windows ? ".exe": ""))
+
+Conda.add("jupyter=1.0.0")
+@test v"4.0.0" == convert(VersionNumber, chomp(readall(`$(joinpath(Conda.SCRIPTDIR, "jupyter-kernelspec")) --version`)))
```

Comment:
The old equivalent to this was ```@test v"4.0.0" == convert(VersionNumber, chomp(readall(`$(joinpath(Conda.scriptsdir("jupyter"), "jupyter-kernelspec")) --version`)))``` which would fail because the script did not work.